Random minor things:
* Add missing UTF-8 encodings to the HTML files.
  https://github.com/rpwoodbu/mosh-chrome/pull/208
* Make things dark-mode esque by default.
  We can improve when CrOS supports this properly.
* Hide the Google DoH knob as it requires a manifest permission.

--- a/mosh_app/mosh_client.html
+++ b/mosh_app/mosh_client.html
@@ -16,12 +16,17 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <head>
+  <meta charset='utf-8'/>
   <title>Mosh Sessions</title>
   <script src="mosh_client.js" type="text/javascript"></script>
   <script src="mosh_version.js" type="text/javascript"></script>
 </head>
 
 <style>
+body {
+  background-color: #111;
+  filter: invert(1);
+}
 
 td {
   white-space:nowrap;
@@ -99,7 +104,7 @@ td {
               </select>
             </td>
           </tr>
-          <tr>
+          <tr hidden>
             <td>
               DNS settings:
               <a
--- a/mosh_app/mosh_window.html
+++ b/mosh_app/mosh_window.html
@@ -16,6 +16,7 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <head>
+  <meta charset='utf-8'/>
   <title>Mosh</title>
   <script src="mosh_manifest.js" type="text/javascript"></script>
   <script src="mosh_window.js" type="text/javascript"></script>
@@ -28,6 +29,7 @@
       height: 100%;
       width: 100%;
       overflow: hidden;
+      background-color: black;
     }
     #terminal {
       display: block;
